test/runtests.jl

```diff
@@ -585,3 +585,7 @@ end
     @test_throws BoundsError insert!(x, 9, true)
     @test x == [1, 1, 10, 99, 2, 3, 1]
 end
+
+@testset "constructor with reftype" begin
+    @test typeof(PooledArray([1 2; 3 4], Int8)) === PooledMatrix{Int, Int8, Matrix{Int8}}
+end
```

Comment:
Maybe put this with similar tests around line 130 above?

Reply:
OK - moved